Cargo.toml

```
@@ -13,6 +13,7 @@ futures = "0.3.4"
 parity-util-mem = { version = "0.7.0", default-features = false, features = ["jemalloc-global"] }
 acala-cli = { path = "cli" }
 acala-service = { path = "service" }
+mandala-runtime = { path = "runtime/mandala" }
```

```suggestion
mandala-runtime = { path = "runtime/mandala", optional = true }
```

We want to be able to build Karura without building mandala.